src/pages/documentation/environment-variables.mdx

```diff
@@ -90,6 +90,10 @@ Strikethrough env vars are deprecated.
 
 #### QR codes
 
+<Callout type="warning">
+  Generating QR Codes through Shlink is being deprecated (see <a href="https://github.com/shlinkio/shlink/issues/2408">#2408</a>).
```

Comment:
I reckon markdown works inside JSX when used in MDX files.

Also, I think it's better to be more specific on where the link is taking you, as an issue number in isolation does not have a lot of meaning in this context.  

```suggestion
  Generating QR codes through Shlink is being deprecated. See [this issue](https://github.com/shlinkio/shlink/issues/2408) for details.
```